packages/client/components/ActivityLibrary/ActivityIllustrations.tsx

```diff
@@ -0,0 +1,115 @@
+import aChristmasCarolRetrospectiveTemplate from '../../../../static/images/illustrations/aChristmasCarolRetrospectiveTemplate.png'
```

Comment:
Extracted a file with all the illustrations to prevent copy paste and having multiple places to update while adding a new template